doc/py_tutorials/py_feature2d/py_brief/py_brief.markdown

```diff
@@ -43,6 +43,10 @@ points than for SURF points.
 In short, BRIEF is a faster method feature descriptor calculation and matching. It also provides
 high recognition rate unless there is large in-plane rotation.
 
+STAR in OpenCV 
```

Comment:
There is this sentence few lines below:

> CenSurE
> detector is called STAR detector in OpenCV

Perhaps, it make sense to use `STAR(CenSurE)` here.